src/main/java/ladder/util/StringUtils.java

```diff
@@ -0,0 +1,8 @@
+package ladder.util;
+
+public class StringUtils {
+
+    public static boolean isBlank(String value) {
+        return (value == null || value.trim().isEmpty()) ? true : false;
```

Comment:
```suggestion
        return value == null || value.trim().isEmpty();
```
삼항연산자의 값이 곧 리턴값이므로 삼항연상이 필요 없어 보여요. 
예를들어 다음도 비슷한 상황일것 같습니다.
```java
if (isEmpty() == true) {
    ...
```